main.yml

```diff
@@ -19,10 +19,20 @@
         script_mode: "{{ mode }}"
       when: mode is defined
 
+    - name: set_fact script_mode
+      set_fact:
+        MULTITENANT_SERVER_VARS_LOCAL_LOCATION: "{{ MULTITENANT_SERVER_VARS }}"
+      when: MULTITENANT_SERVER_VARS is defined
+
+    - name: set_fact script_mode
+      set_fact:
+        server_vars_path: "{{ MULTITENANT_SERVER_VARS }}"
+      when: MULTITENANT_SERVER_VARS is defined
+
```

Comment:
@sidorovdmitry 
because of playbook multifunctionality. We cannot use server_vars_path variable herehttps://github.com/raccoongang/themex_script/blob/7d73774147f1b6ee6bb6d2eb3df99e71067d3497/roles/prepare-venv/tasks/main.yml#L13